examples/whisper.wasm/index-tmpl.html

```diff
@@ -24,6 +24,8 @@
                 overflow-x: scroll;
             }
         </style>
+        <script src="coi-serviceworker.js"></script>
```

Comment:
Since this script is shared by all the WASM examples, it can be moved from `examples/whisper.wasm/` to `examples/`, similar to how `helpers.js` is already there.

Reply:
Ah yes, good point. I'll move it.